lib/launcher.rb

```diff
@@ -102,13 +102,19 @@ def https_app(https_port, plugins = https_plugins)
         :SSLVerifyClient => OpenSSL::SSL::VERIFY_PEER,
         :SSLPrivateKey => load_ssl_private_key(settings.ssl_private_key),
         :SSLCertificate => load_ssl_certificate(settings.ssl_certificate),
-        :SSLCACertificateFile => settings.ssl_ca_file,
+        :SSLCACertificateFile => verify_ssl_ca_file(settings.ssl_ca_file),
         :SSLOptions => ssl_options,
         :SSLCiphers => CIPHERS - Proxy::SETTINGS.ssl_disabled_ciphers,
         :daemonize => false,
       }
     end
 
+    def verify_ssl_ca_file(path)
+     return path if File.readable?(path)
+
+     logger.error "Unable to load CA SSL certificate. Are the values correct in settings.yml and do permissions allow reading?"
```

Comment:
Author of the original issue expected an explicit error in the logs, but letting the service start. I'd be inclined to raise an exception the same way as we do when private key or certificate are invalid

Reply:
I would probably avoid such naming for X509 cert, this can make impression that the method actually validates the cert while I read the code as it checks if file is readable.

I also doubt if `File.readable?` really needs an explicit function, I would prefer doing this before the return statement of the previous method, but this is just a preference.

Also the wording is a bit rough, I would also add the filepath in the error message too: "Unable to read #{path}, check settings.yml and permissions" or something like that.

Reply:
To elaborate "naming", what I mean is the name of the method.